packages/docs-gesture-handler/src/theme/Footer/index.js

```
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { Footer as TRexFooter } from '@swmansion/t-rex-ui';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)